### ISSUE TITLE
The CI is red due to helix error

### ISSUE THREAD

**Issue by adamsitnik** [dotnet/performance#240]
I just realized that the [CI](https://dev.azure.com/dnceng/public/_build?definitionId=271&_a=summary) is red due to helix error.

Sample error from the [logs](https://dev.azure.com/dnceng/public/_build/results?buildId=79462):

```log
2019-01-24T15:09:06.0404888Z ##[section]Starting: Send job to Helix (Windows)
2019-01-24T15:09:06.0410392Z ==============================================================================
2019-01-24T15:09:06.0410583Z Task         : PowerShell
2019-01-24T15:09:06.0410940Z Description  : Run a PowerShell script on Windows, macOS, or Linux.
2019-01-24T15:09:06.0411092Z Version      : 2.140.2
2019-01-24T15:09:06.0411239Z Author       : Microsoft Corporation
2019-01-24T15:09:06.0411396Z Help         : [More Information](https://go.microsoft.com/fwlink/?LinkID=613736)
2019-01-24T15:09:06.0411573Z ==============================================================================
2019-01-24T15:09:08.2222116Z Generating script.
2019-01-24T15:09:08.5073308Z ##[command]"C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe" -NoLogo -NoProfile -NonInteractive -ExecutionPolicy Unrestricted -Command ". 'D:\a\_temp\ab9661ca-df8d-430f-8ef0-f51e79061f1a.ps1'"
2019-01-24T15:09:11.2325950Z dotnet-install: Downloading link: https://dotnetcli.azureedge.net/dotnet/Sdk/2.1.403/dotnet-sdk-2.1.403-win-x64.zip
2019-01-24T15:09:12.8531767Z dotnet-install: Extracting zip from https://dotnetcli.azureedge.net/dotnet/Sdk/2.1.403/dotnet-sdk-2.1.403-win-x64.zip
2019-01-24T15:09:20.7478695Z dotnet-install: Adding to current process PATH: "D:\a\1\s\.dotnet\". Note: This change will not be visible if PowerShell was run as a child process.
2019-01-24T15:09:20.7507728Z dotnet-install: Installation finished
2019-01-24T15:09:21.8657504Z D:\a\1\s\.dotnet\sdk\2.1.403\MSBuild.dll /nologo -maxcpucount /m -verbosity:m /v:minimal /bl:D:\a\1\s\artifacts\log\x64_CoreCLR_netcoreapp3.0\SendToHelix.binlog /clp:Summary /nr:True /p:TreatWarningsAsErrors=true /warnaserror D:\a\1\s\eng\common\helixpublish.proj
2019-01-24T15:09:29.3443653Z C:\Users\VssAdministrator\.nuget\packages\microsoft.dotnet.helix.sdk\1.0.0-beta.18629.1\tools\Microsoft.DotNet.Helix.Sdk.MonoQueue.targets(85,5): error : HttpOperationException: Operation returned an invalid status code 'BadRequest' [D:\a\1\s\eng\common\helixpublish.proj]
2019-01-24T15:09:29.3444375Z C:\Users\VssAdministrator\.nuget\packages\microsoft.dotnet.helix.sdk\1.0.0-beta.18629.1\tools\Microsoft.DotNet.Helix.Sdk.MonoQueue.targets(85,5): error :    at Microsoft.DotNet.Helix.Client.Job.NewOperationWithHttpMessagesAsync(JobCreationRequest newJob, Dictionary`2 customHeaders, CancellationToken cancellationToken) in /_/src/Microsoft.DotNet.Helix/Client/CSharp/Generated/Job.cs:line 256 [D:\a\1\s\eng\common\helixpublish.proj]
2019-01-24T15:09:29.3446344Z C:\Users\VssAdministrator\.nuget\packages\microsoft.dotnet.helix.sdk\1.0.0-beta.18629.1\tools\Microsoft.DotNet.Helix.Sdk.MonoQueue.targets(85,5): error :    at Microsoft.DotNet.Helix.Client.JobExtensions.NewOperationAsync(IJob operations, JobCreationRequest newJob, CancellationToken cancellationToken) in /_/src/Microsoft.DotNet.Helix/Client/CSharp/Generated/JobExtensions.cs:line 66 [D:\a\1\s\eng\common\helixpublish.proj]
2019-01-24T15:09:29.3446911Z C:\Users\VssAdministrator\.nuget\packages\microsoft.dotnet.helix.sdk\1.0.0-beta.18629.1\tools\Microsoft.DotNet.Helix.Sdk.MonoQueue.targets(85,5): error :    at Microsoft.DotNet.Helix.Client.HelixApi.RetryAsync[T](Func`1 function, Action`1 logRetry, Func`2 isRetryable) [D:\a\1\s\eng\common\helixpublish.proj]
2019-01-24T15:09:29.3447794Z C:\Users\VssAdministrator\.nuget\packages\microsoft.dotnet.helix.sdk\1.0.0-beta.18629.1\tools\Microsoft.DotNet.Helix.Sdk.MonoQueue.targets(85,5): error :    at Microsoft.DotNet.Helix.Client.JobDefinition.SendAsync(Action`1 log) in /_/src/Microsoft.DotNet.Helix/JobSender/JobDefinition.cs:line 153 [D:\a\1\s\eng\common\helixpublish.proj]
2019-01-24T15:09:29.3448424Z C:\Users\VssAdministrator\.nuget\packages\microsoft.dotnet.helix.sdk\1.0.0-beta.18629.1\tools\Microsoft.DotNet.Helix.Sdk.MonoQueue.targets(85,5): error :    at Microsoft.DotNet.Helix.Sdk.SendHelixJob.ExecuteCore() in /_/src/Microsoft.DotNet.Helix/Sdk/SendHelixJob.cs:line 209 [D:\a\1\s\eng\common\helixpublish.proj]
2019-01-24T15:09:29.3449630Z C:\Users\VssAdministrator\.nuget\packages\microsoft.dotnet.helix.sdk\1.0.0-beta.18629.1\tools\Microsoft.DotNet.Helix.Sdk.MonoQueue.targets(85,5): error :    at Microsoft.DotNet.Helix.Sdk.HelixTask.Execute() in /_/src/Microsoft.DotNet.Helix/Sdk/HelixTask.cs:line 43 [D:\a\1\s\eng\common\helixpublish.proj]
2019-01-24T15:09:29.3451665Z C:\Users\VssAdministrator\.nuget\packages\microsoft.dotnet.helix.sdk\1.0.0-beta.18629.1\tools\Microsoft.DotNet.Helix.Sdk.MonoQueue.targets(85,5): error :  [D:\a\1\s\eng\common\helixpublish.proj]
2019-01-24T15:09:29.3740426Z 
2019-01-24T15:09:29.3741204Z Build FAILED.
2019-01-24T15:09:29.3741446Z 
2019-01-24T15:09:29.3742297Z C:\Users\VssAdministrator\.nuget\packages\microsoft.dotnet.helix.sdk\1.0.0-beta.18629.1\tools\Microsoft.DotNet.Helix.Sdk.MonoQueue.targets(85,5): error : HttpOperationException: Operation returned an invalid status code 'BadRequest' [D:\a\1\s\eng\common\helixpublish.proj]
2019-01-24T15:09:29.3742997Z C:\Users\VssAdministrator\.nuget\packages\microsoft.dotnet.helix.sdk\1.0.0-beta.18629.1\tools\Microsoft.DotNet.Helix.Sdk.MonoQueue.targets(85,5): error :    at Microsoft.DotNet.Helix.Client.Job.NewOperationWithHttpMessagesAsync(JobCreationRequest newJob, Dictionary`2 customHeaders, CancellationToken cancellationToken) in /_/src/Microsoft.DotNet.Helix/Client/CSharp/Generated/Job.cs:line 256 [D:\a\1\s\eng\common\helixpublish.proj]
2019-01-24T15:09:29.3744009Z C:\Users\VssAdministrator\.nuget\packages\microsoft.dotnet.helix.sdk\1.0.0-beta.18629.1\tools\Microsoft.DotNet.Helix.Sdk.MonoQueue.targets(85,5): error :    at Microsoft.DotNet.Helix.Client.JobExtensions.NewOperationAsync(IJob operations, JobCreationRequest newJob, CancellationToken cancellationToken) in /_/src/Microsoft.DotNet.Helix/Client/CSharp/Generated/JobExtensions.cs:line 66 [D:\a\1\s\eng\common\helixpublish.proj]
2019-01-24T15:09:29.3744919Z C:\Users\VssAdministrator\.nuget\packages\microsoft.dotnet.helix.sdk\1.0.0-beta.18629.1\tools\Microsoft.DotNet.Helix.Sdk.MonoQueue.targets(85,5): error :    at Microsoft.DotNet.Helix.Client.HelixApi.RetryAsync[T](Func`1 function, Action`1 logRetry, Func`2 isRetryable) [D:\a\1\s\eng\common\helixpublish.proj]
2019-01-24T15:09:29.3745553Z C:\Users\VssAdministrator\.nuget\packages\microsoft.dotnet.helix.sdk\1.0.0-beta.18629.1\tools\Microsoft.DotNet.Helix.Sdk.MonoQueue.targets(85,5): error :    at Microsoft.DotNet.Helix.Client.JobDefinition.SendAsync(Action`1 log) in /_/src/Microsoft.DotNet.Helix/JobSender/JobDefinition.cs:line 153 [D:\a\1\s\eng\common\helixpublish.proj]
2019-01-24T15:09:29.3746233Z C:\Users\VssAdministrator\.nuget\packages\microsoft.dotnet.helix.sdk\1.0.0-beta.18629.1\tools\Microsoft.DotNet.Helix.Sdk.MonoQueue.targets(85,5): error :    at Microsoft.DotNet.Helix.Sdk.SendHelixJob.ExecuteCore() in /_/src/Microsoft.DotNet.Helix/Sdk/SendHelixJob.cs:line 209 [D:\a\1\s\eng\common\helixpublish.proj]
2019-01-24T15:09:29.3746773Z C:\Users\VssAdministrator\.nuget\packages\microsoft.dotnet.helix.sdk\1.0.0-beta.18629.1\tools\Microsoft.DotNet.Helix.Sdk.MonoQueue.targets(85,5): error :    at Microsoft.DotNet.Helix.Sdk.HelixTask.Execute() in /_/src/Microsoft.DotNet.Helix/Sdk/HelixTask.cs:line 43 [D:\a\1\s\eng\common\helixpublish.proj]
2019-01-24T15:09:29.3747256Z C:\Users\VssAdministrator\.nuget\packages\microsoft.dotnet.helix.sdk\1.0.0-beta.18629.1\tools\Microsoft.DotNet.Helix.Sdk.MonoQueue.targets(85,5): error :  [D:\a\1\s\eng\common\helixpublish.proj]
2019-01-24T15:09:29.3747585Z     0 Warning(s)
2019-01-24T15:09:29.3747818Z     1 Error(s)
2019-01-24T15:09:29.3748007Z 
2019-01-24T15:09:29.3748217Z Time Elapsed 00:00:07.43
2019-01-24T15:09:29.4480672Z Build failed.
2019-01-24T15:09:29.4574864Z See log: D:\a\1\s\artifacts\log\x64_CoreCLR_netcoreapp3.0\SendToHelix.binlog
2019-01-24T15:09:29.7796077Z ##[error]PowerShell exited with code '1'.
2019-01-24T15:09:29.8219479Z ##[section]Finishing: Send job to Helix (Windows)
```

@adiaaida could you please take a look? 

/cc @billwert @brianrob @jorive

**Comment by adiaaida** [comment]
I currently do not have a functional Windows box, but once I get my VM running, I will take a look.

**Comment by adamsitnik** [comment]
Closed by #243 

@adiaaida once again thanks for the explanation!
